### PR TITLE
inital commit

### PR DESCRIPTION
**Pull request by MeetQingLiu**
SAS Syntax and Theme mimic The SAS Sytem syntax and highlight, modified based on the SAS programming package

### CODE REVIEW

**Comment by MeetQingLiu** [comment]
changed

**Comment by AlexKvazos** [comment]
Hello @MeetQingLiu, you can run this tests locally instead of having to reopen the issue each push. It will be more agile for you and less noisy for us!

https://github.com/packagecontrol/st_package_reviewer

**Comment by wbond** [comment]
In about an hour when the rate limit has been cleared, you can use https://packagecontrol.io/test_repo to submit your JSON for testing. It will produce 100% the same results as packagecontrol-bot will produce.

**Comment by MeetQingLiu** [comment]
Hi Wbond,
@wbond ,

> The Error: The binding ['ctrl+e'] unconditionally overrides a default binding File: Default.sublime-keymap 

always exist even if I delete the default binding file.
It seems that the packgage cotrol system don't update my package simultaneously.
Do you have any suggestions?
Thanks .

**Comment by FichteFoll** [comment]
@MeetQingLiu the reviewer analyzes your package based on your latest release tag. You haven't pushed a new tag for your commits yet, which is why it always checks against https://github.com/MeetQingLiu/SAS-Syntax-and-Theme/tree/v1.0.0.

**Comment by MeetQingLiu** [comment]
OK!!! @FichteFoll , Thank you for your help!

File: repository/s.json [hunk 1]
```diff
@@ -106,6 +106,19 @@
...
 		},
 		{
+			"name": "SAS Syntax and Theme",
+			"description": "SAS Syntax and Theme mimic The SAS Sytem syntax and highlight" ,
+			"details": "https://github.com/MeetQingLiu/SAS-Syntax-and-Theme",
+			"labels": ["language syntax", "Theme", "build-system", "snippets"],
+			"author": ["Qing"],
+			"releases": [
+				{
+					"sublime_text": "*",
```
Comment (FichteFoll) [inline repository/s.json]: You need to change this to `"sublime_text": ">=3092",`.

**Pull request merged**